AudibleApi.Common/RecordDto.cs

```diff
@@ -0,0 +1,129 @@
+﻿using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace AudibleApi.Common
+{
+	public partial class RecordDto : DtoBase<RecordDto>
+	{
+		[JsonProperty("md5")]
+		public string Md5 { get; set; }
+
+		[JsonProperty("payload")]
+		public Payload Payload { get; set; }
+	}
+
+	public partial class Payload
+	{
+		[JsonProperty("acr")]
+		public string Acr { get; set; }
+
+		[JsonProperty("records")]
+		[JsonConverter(typeof(RecordConverter))]
+		public List<IRecord> Records { get; set; }
+
+		[JsonProperty("guid")]
+		public string Guid { get; set; }
+
+		[JsonProperty("key")]
+		public string Asin { get; set; }
+
+		[JsonProperty("type")]
+		public string Type { get; set; }
+	}
+
+	internal class RecordConverter : JsonConverter
+	{
+		public override bool CanConvert(Type typeToConvert) => false;
+
+		public override object ReadJson(JsonReader reader, Type objectType, object existingValue, JsonSerializer serializer)
+		{
+			static IRecord createRecord(JObject jObj)
+			{
+				var type = jObj.Value<string>("type");
+				var creationTime = DateTime.Parse(jObj.Value<string>("creationTime")).ToLocalTime();
```

Comment:
creationTime and lastModificationTime are UTC strings with no time zone.

Reply:
Sorry, I'll never remember this. Can you add an in-line comment?

Reply:
done

Reply:
Gracias